src/lib.rs

```diff
@@ -230,6 +250,7 @@ pub struct EdgeRef<'a, NodeId, EdgeId> {
     pub from: &'a NodeId,
     pub to: &'a NodeId,
     pub id: &'a EdgeId,
+    pub edge_type: &'a EdgeType,
```

Comment:
Should be EdgeData

Reply:
Why? If we use `EdgeData` here we are locking away the possibility to override the `EdgeData` with something simpler/richer which also _contains_ the `EdgeType`. If you remember from our previous discussions we settled on having `EdgeType` and `NodeType` being the lowest common denominator between the registry & osrank, but we also discussed how it's nice for osrank to be able to have full control on how to map this data.

My vote would be to leave `EdgeType` here, as it's literally all we need 😉

Reply:
ps. Note how we cannot use `G::EdgeData` as we don't have a `G` in scope, alas, and due to the fact `EdgeType` was all I needed .. this is what I have added 😅